torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3309,10 +3339,11 @@ def _maybe_evaluate_static(
                 continue
             subst = {}
             for ra in self.deferred_runtime_asserts.get(s, ()):
+                # NB: simplify here is load bearing, as it ensures that we
+                # apply replacements from unbacked_replacements
```

Comment:
what's `unbacked_replacements`, did you mean `eliminated_unbacked`?

Reply:
Yes, sort of, this comment is just totally out of date

Reply:
Essentially, we need to make sure we apply replacements that were previously impeded by resolve_unbacked=False. So actually, it's like all unbacked symbol replacements that are NOT eliminate unbacked